editor/src/components/canvas/canvas-strategies/strategies/move-reorder-metastrategy.tsx

```diff
@@ -0,0 +1,109 @@
+import { mapDropNulls } from '../../../../core/shared/array-utils'
+import { ImmediateParentBounds } from '../../controls/parent-bounds'
+import { ImmediateParentOutlines } from '../../controls/parent-outlines'
+import {
+  DragOutlineControl,
+  dragTargetsElementPaths,
+} from '../../controls/select-mode/drag-outline-control'
+import { CanvasStrategyFactory, MetaCanvasStrategy } from '../canvas-strategies'
+import {
+  CustomStrategyState,
+  getTargetPathsFromInteractionTarget,
+  InteractionCanvasState,
+  CanvasStrategy,
+  emptyStrategyApplicationResult,
+  controlWithProps,
+} from '../canvas-strategy-types'
+import { InteractionSession } from '../interaction-state'
+import { absoluteMoveStrategy } from './absolute-move-strategy'
+import { flexReorderStrategy } from './flex-reorder-strategy'
+import { flowReorderStrategy } from './flow-reorder-strategy'
+import { relativeMoveStrategy } from './relative-move-strategy'
+import { reparentMetaStrategy } from './reparent-metastrategy'
+import { getDragTargets } from './shared-move-strategies-helpers'
+
+// + ancestor strategy !!???
+const baseMoveStrategyFactories: Array<CanvasStrategyFactory> = [
+  absoluteMoveStrategy,
+  flexReorderStrategy,
+  flowReorderStrategy,
+  relativeMoveStrategy,
+]
+
+export const moveReorderMetaStrategy: MetaCanvasStrategy = (
```

Comment:
it sounds like this is all drag-to-move strategies: move, reorder, reparent. I don't think this is too much at all! actually, I think this is a pretty elegant solution.

```suggestion
export const dragToMoveMetaStrategy: MetaCanvasStrategy = (
```